guides/v2.1/cloud/project/new-relic.md

```diff
@@ -138,9 +138,11 @@ We recommend reviewing tracked data:
 
 All collected data details the time spent on an action transmitting data, queries, or _Redis_ data. If queries cause issues, New Relic provides information to track and respond to it.
 
+To learn more about using and customizing New Relic services for your project, see [New Relic Infrastructure documentation](https://docs.newrelic.com/docs/infrastructure/new-relic-infrastructure/getting-started/introduction-new-relic-infrastructure)
+
 ## New Relic and Blackfire
 
-You can use the high-level data captured by New Relic APM along with the code-level data captured by the [Blackfire Profiler]({{ page.baseurl }}/cloud/project/project-integrate-blackfire.html) to get a comprehensive view of your Magento store performance.
+You can use the high-level data captured by New Relic APM to see what is happening in your application in realtime. The New Relic Insights helps to focus on events, transactions, regions, etcetera, to correlate data.  Also, with the code-level data captured by the [Blackfire Profiler]({{ page.baseurl }}/cloud/project/project-integrate-blackfire.html) you get a comprehensive view of your Magento store performance.
```

Comment:
```suggestion
You can use the high-level data captured by New Relic APM to see what is happening in your application in realtime. The analytics from New Relic Insights help to focus on events, transactions, regions, etcetera, to correlate data.  Also, with the code-level data captured by the [Blackfire Profiler]({{ page.baseurl }}/cloud/project/project-integrate-blackfire.html) you get a comprehensive view of your Magento store performance.
```

Reply:
Removed this statement. Insights is not supposed to be here.